dashboard/test/models/contact_rollups_final_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class ContactRollupsFinalTest < ActiveSupport::TestCase
+  test 'overwrite_from_processed_table! overwrites contact_rollups_final' do
+    clean_tables
+
+    processed = create_list :contact_rollups_processed, 5
+    create_list :contact_rollups_final, 3
+
+    ContactRollupsFinal.delete_all # Stand in for ContactRollupsFinal.truncate, which tests don't allow b/c modifies DDL
```

Comment:
Is it possible to stub `truncate` function instead?

Reply:
I'm not sure what is best here -- I could just not create any rows in `contact_rollups_final`, otherwise I need to do some sort of truncate/delete_all.

Reply:
I ended up making the test explicitly test the insert method -- I can't figure out how to get truncate to run from tests.